scripts/userstyles.yml

```
@@ -905,6 +907,14 @@ userstyles:
     readme:
       app-link: https://app.raindrop.io
     current-maintainers: [*thismoon]
+  react:
+    name: React
```

```suggestion
  react.dev:
    name: React.dev
```

I think this naming will clarify that this port is for React's website/documentation and not for actual use in React code. You will need to rename the `styles/react` directory to `styles/react.dev`, and update the metadata in the comment at the top of the `catppuccin.user.less` file (`deno task lint` should tell you exactly what needs to change).